docs/fr/mkdocs.yml

```diff
@@ -54,6 +54,7 @@ nav:
 - features.md
 - fastapi-people.md
 - python-types.md
+- history-design-future.md
```

Comment:
In the English doc, `history-design-future` comes between `alternatives.md` and `external-links.md`.